src/components/Alert/README.md

```diff
@@ -0,0 +1,55 @@
+Alert
```

Comment:
No need for this title here. It will duplicate like so:
<img width="390" alt="screen shot 2018-01-19 at 16 57 38" src="https://user-images.githubusercontent.com/2573602/35166958-d7be2032-fd3a-11e7-8365-1f8c30778f82.png">

The "Button" before "Default"